packages/neuron-ui/src/components/WalletEditor/index.tsx

```diff
@@ -92,7 +92,8 @@ export default (props: React.PropsWithoutRef<ContentProps & RouteComponentProps<
           wallet={myWallet}
           dispatch={dispatch}
           checkType={CheckType.EditWallet}
-          handle={() => props.history.push(`${Routes.SettingsWallets}`)}
+          // handle={() => props.history.push(`${Routes.SettingsWallets}`)}
```

Comment:
Delete this instead of comment out as well.

Reply:
fixed